api/maps/MarkerInfoWindow.tsx

```diff
@@ -87,8 +89,16 @@ export const MarkerInfoWindow = ({
   useEffect(() => {
     if (marker && clusterer) {
       clusterer.addMarker(marker);
+      markerMap.set(projectId, marker as unknown as google.maps.Marker);
```

Comment:
this bypasses the type-checking system but may cause runtime issues bc of compatibility. `marker` = AdvancedMarkerElement but `markerMap` takes in google.maps.Marker.

Reply:
hmmm, I don't want to run the risk of runtime errors and when I checked it on my VSCode it did say Marker is depreciated. Since this feature does not need to be merged in ASAP, let's work together on a fix after Thanksgiving Break.